trino/Dockerfile

```diff
@@ -4,56 +4,62 @@
 # Ignoring DL4006 globally because we inherit the SHELL from our base image
 # hadolint global ignore=DL3038,DL4006
 
-FROM stackable/image/java-base
+FROM stackable/image/java-devel AS storage-connector-builder
```

Comment:
```suggestion
# Not tagging base image because it is built as part of the same process
# hadolint ignore=DL3006
FROM stackable/image/java-devel AS storage-connector-builder
```